packages/celotool/src/e2e-tests/transfer_tests.ts

```diff
@@ -666,9 +666,9 @@ describe('Transfer tests', function(this: any) {
               await inflationManager.changeInflationFactorOnNextTransfer(new BigNumber(2))
 
               const intrinsicGas = 155000
-              const gas = intrinsicGas + 1000
+              const gas = intrinsicGas + 5000
               const txRes = await runTestTransaction(
-                await transferCeloGold(FromAddress, ToAddress, TransferAmount, {
+                await transferCeloDollars(FromAddress, ToAddress, TransferAmount, {
```

Comment:
Why did this change?

Reply:
because now the gold transfer only requires the intrinsic gas to succeed, so it's not possible to test a transfer > intrinsic gas but < required.

Reply:
no longer applies!